app/src/main/res/layout/event_layout.xml

```diff
@@ -17,7 +17,7 @@
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:background="?attr/selectableItemBackground"
+        android:background="@{ event.state.equals(`draft`) ? @color/event_state_draft : android.R.attr.selectableItemBackground }"
```

Comment:
This will cause ripple effect to be disabled

Reply:
I have made changes. Please review.